Source/Orts.Simulation/Simulation/RollingStocks/MSTSDieselLocomotive.cs

```diff
@@ -990,6 +990,12 @@ public string GetDPDebugStatus()
                     {
                         data = (data / MaxDynamicBrakeForceN) * DynamicBrakeMaxCurrentA;
                     }
+                    if (ThrottlePercent == 0 && data > 0)
+                    {
+                        data = DynamicBrakePercent == -1? (data / MaxForceN) * MaxCurrentA
+                            : DynamicBrakePercent == 0? (data / MaxDynamicBrakeForceN) * DynamicBrakeMaxCurrentA
+                            : data;
+                    }
```

Comment:
```suggestion
                    if (ThrottlePercent >= 0 && DynamicBrakePercent == -1)
                    {
                        data = (data / MaxForceN) * MaxCurrentA;
                    }
                    if (ThrottlePercent == 0 && DynamicBrakePercent >= 0)
                    {
                        data = (data / MaxDynamicBrakeForceN) * DynamicBrakeMaxCurrentA;
                    }
```

Reply:
The above suggestion should replace all of the if conditions (you can't make comments on non edited lines currently in Github).

Reply:
I wanted to make this suggestion in order to simplify the code and avoid a case where you get data = data.

Reply:
You are referring to all the **if conditions** inside the `case CABViewControlUnits.AMPS:` should be  replaced by the suggested code.

Reply:
Yes

Reply:
Fix improved, with your code. 
Thank you.